### ISSUE TITLE
More verbose output in `coh test`

### ISSUE THREAD

**Issue by johnslavik** [coherent-oss/coherent.cli#8]
Currently, the `test` command is very slow and silent.
Silent to the point that I'm wondering if it's actually doing what it is supposed to.
Let's give it a more verbose output, informing about each step being performed.
E.g.
```
> coh test
Bootstrapping pyproject.toml... done.
Installing dependencies... done.
Running tests...
<test output>
```

**Comment by jaraco** [comment]
Yep. You've identified the two key areas of slowness. We'll want to speed those up eventually, but in the meantime, some output would be nice.
